_test/test_sqw_pixels/test_PixelAlignment.m

```diff
@@ -47,13 +76,13 @@ function test_finalize_alignment_membacked(~)
             al_matr = rotvec_to_rotmat2([0, pi/4, pi/4]);
 
             pdm.alignment_matr = al_matr;
-            assertTrue(pdm.is_misaligned);
+            assertFalse(pdm.is_misaligned);
```

Comment:
for this, assigning non-eye alignment_matr makes misaligned false as presumably the alignment is assigned by the set. However, above line 33 assigning non-eye makes misaligned true. There seems to be an inconsistency here. Even if this is explicable, it makes the use of setting alignment problematic for the average user

Reply:
No inconsistency. In one situation, asignment done for memory-based object which is aligned on-fly. 
For other case, it is filebacked object and aligning it is presumably takes a long time. To avoid this, we make it misaligned and will perform alignment as side-operation along with other operations.